### PR TITLE
Cap event description length at 2000 chars.

### PR DESCRIPTION
**Pull request by alloy**
Forgot to cap the length.

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@ class Event < ApplicationRecord
   friendly_id :name, use: [:slugged, :finders]
   has_many :rsvps
 
+  validates :description, length: { maximum: 2000 }
```
Comment (mzikherman) [inline app/models/event.rb:6]: I'm guessing there are none that need to be migrated to be made valid?

Comment (alloy) [reply app/models/event.rb:6]: Nope, this field was added with the previous PR.

Comment (mzikherman) [reply app/models/event.rb:6]: Should this be a DB constraint as well with a max length? If so, (I'm being so pedantic I'm sorry), the field should be switched from `TEXT` (cannot be bounded) to a string (`VARCHAR` which can be bounded to 2000 and not just 255). Still...so #minor

Comment (alloy) [reply app/models/event.rb:6]: Aye, you’re completely correct of course 👌 

Seeing as this is such a simple app for now and it’s already deployed, I think we’re ok to leave it as-is.

**Pull request merged**